incense/experiment_loader.py

```diff
@@ -17,8 +17,8 @@
 class ExperimentLoader:
     """Loads artifacts related to experiments."""
 
-    def __init__(self, mongo_uri=None, db_name="sacred", unpickle: bool = True):
-        client: MongoClient = MongoClient(mongo_uri)
+    def __init__(self, mongo_uri=None, db_name="sacred", unpickle: bool = True, **docker_client_kwargs):
```

Comment:
```suggestion
    def __init__(self, mongo_uri=None, db_name="sacred", unpickle: bool = True, **mongo_client_kwargs):
```

Sounds more reasonable.